.agignore

```diff
@@ -0,0 +1 @@
+dist
```

Comment:
I think ack disappeared from my system in an OS upgrade or something... and I tried `ag` after remembering I think @theengineear mentioned it a while back. I was never really bothered by `ack` but `ag` does seem to live up to its claim of speed. Otherwise it seems pretty similar but I'm pretty basic in how I use these tools.